docs/content/blog/v1-alpha3.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "1.0.0-alpha.3 is here!"
+description: "Check out the new features such as MongoDB, namespaces, bundle state and credential import."
+date: "2021-09-21"
+authorname: "Carolyn Van Slyck"
+author: "@carolynvs"
+authorlink: "https://carolynvanslyck.com/"
+authorimage: "https://github.com/carolynvs.png"
+tags: ["release-notes", "v1"]
+---
+
+Check out the new features introduced in Porter v1.0.0-alpha.3 such as MongoDB, namespaces, bundle state and credential import.
+<!--more-->
+
+This is a big release for our v1 alpha with LOTS of major changes that we hope make Porter easier to use such as [MongoDB](#mongodb-support), [namespaces](#namespaces), [labels](#labels), [bundle state](#bundle-state), and [credential import](#import-credentials-and-parameter-sets). Our [release notes] have a full accounting of all **fifty-eight** pull requests that were included in this release.
+
+A v1 version of our documentation is available at https://release-v1.porter.sh.
+
+[release notes]: https://github.com/getporter/porter/releases/tag/v1.0.0-alpha.3
+
+## Breaking Changes
+Safety first! In this release we have removed support for deprecated flags such as --tag and fields like tag in porter.yaml.
+If you have been ignoring a warning message in Porter's output for months, now is the time to fix those messages before upgrading.
+
+## MongoDb Support
+Porter now stores its data in MongoDB! We no longer store installation records on the filesystem, though we still do have a ~/.porter directory with our configuration and cache.
+In dev and test, Porter by default runs MongoDB in a container for you on porter 27018 and connects to it.
```

Comment:
```suggestion
In dev and test, Porter by default runs MongoDB in a container for you on port 27018 and connects to it.
```

But I kinda like the typo 😉

Reply:
omg I make this typo EVERYWHERE now 🤦‍♀️